src/olympia/blocklist/utils.py

```diff
@@ -17,8 +17,8 @@ def add_version_log_for_blocked_versions(obj, al):
     from olympia.activity.models import VersionLog
 
     VersionLog.objects.bulk_create([
-        VersionLog(activity_log=al, version_id=id_chan[0])
-        for version, id_chan in obj.addon_versions.items()
+        VersionLog(activity_log=al, version_id=id_)
+        for version, (id_, _) in obj.addon_versions.items()
```

Comment:
I don't really understand what this change is about.

Reply:
ah, that was a drive-by fix.  I originally looked at changing how `Block.addon_versions` was populated and was checking where it was used and decided expanding the tuples directly was better coding practise (bad past-me!).  I didn't end up changing `addon_versions` because it impacted too many other things but left this fix in.